Cirrious.FluentLayout/AdvancedFluentLayoutExtensions.cs

```diff
@@ -56,14 +59,14 @@ public static FluentLayout WithSameTop(this UIView view, UIView previous)
             return view.Top().EqualTo().TopOf(previous);
         }
 
-        public static FluentLayout WithSameCenterX(this UIView view, UIView previous)
+        public static FluentLayout WithSameCenterX(this UIView view, UIView previous, nfloat? margin = null)
         {
```

Comment:
Well there is already a method call Plus() which does the same thing. Other advanced methods already have a margin parameter but the caller could have added Plus() instead. So it's good to have it here also.